src/toil/worker.py

```diff
@@ -274,13 +274,15 @@ def badWorker():
                     #Get the next block function and list that will contain any messages
                     blockFn = fileStore._blockFn
                     messages = fileStore.loggingMessages
-                    
-                    #Run the job
-                    job._execute( jobWrapper=jobWrapper,
-                                  stats=elementNode if config.stats else None, 
-                                  localTempDir=localTempDir, jobStore=jobStore,
-                                  fileStore=fileStore)
-                    
+                    #Load and run the job
+                    runnable = Job._loadJob(jobWrapper.command, jobStore)
```

Comment:
I'd rather not rename `job` to `runnable` since calling it that might not be consistent with the rest of the code.